src/browser/modules/D3Visualization/lib/visualization/renders/menu.coffee

```diff
@@ -115,6 +115,27 @@ do ->
     onTick: noop
   )
 
+  donutDeleteNode = new neo.Renderer(
+    onGraphChange: (selection, viz) -> createMenuItem(selection, viz, 'nodeDelete', 4, 'delete_node', [0, 0], '\uf1f8', 'Permanently delete node from graph')
+
+    onTick: noop
+  )
+
+  donutConnectNode = new neo.Renderer(
+    onGraphChange: (selection, viz) -> createMenuItem(selection, viz, 'nodeConnect', 5, 'connect_node', [0, 0], '\uf060', 'Draw connection from here')
+
+    onTick: noop
+  )
+
+  donutEditNode = new neo.Renderer(
```

Comment:
<img width="206" alt="oskarhane-mbpt 2018-08-17 at 11 21 37" src="https://user-images.githubusercontent.com/849508/44258700-7ff47c80-a207-11e8-9f54-00db5cc4d2c4.png">
The icons are no longer correctly aligned.  The 6th param for `createMenuItem` allows offsetting of the icons. 

How do the icons look when a node size is changed?